ae_gpcam/bluesky_config/scripts/roi_reduction_consumer.py

```diff
@@ -131,6 +148,37 @@ def compute_peak_area(Q, I, q_start, q_stop):
     return np.sum((data_section - background) * dQ)
 
 
+def parse_name(inp):
```

Comment:
Could a few example names be added here in a comment?

Reply:
I would rather not as hopefully we will not have these strings going forward ;)